src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -0,0 +1,142 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class Lights extends SubsystemBase {
+  private LightCodes currentLightStatus = LightCodes.OFF;
+  private AddressableLED m_led = new AddressableLED(RobotMap.LED_PWM_PORT);
+  private AddressableLEDBuffer m_ledBuffer;
```

Comment:
Let's not use this `m_` convention and just call things what they are in camelCase